tensorboard/data/server/pip_package/build.py

```diff
@@ -37,8 +37,8 @@ def main():
     if cpu_name == "arm64":
         mac_platform = "macosx_11_0"
     platform_name = {
-        # using platform tag values from TensorFlow releases
-        "Linux": "manylinux2014",
+        # We build on an Ubuntu 20.04 image, which ships with glibc 2.31.
```

Comment:
Is there any link we can add here that verifies this?

Don't worry too much if that's not the case or you can't find one easily, but it would be nice to have some sort of source of truth for this.

Reply:
Added a link to the glibc source shipped with 20.04 (focal fossa) https://launchpad.net/ubuntu/focal/+source/glibc